Source/JavaScriptCore/runtime/WeakGCMap.h

```diff
@@ -103,6 +104,17 @@ class WeakGCMap final : public WeakGCHashTable {
 
     void pruneStaleEntries() final;
 
+    template<typename Func>
+    void forEach(Func func)
```

Comment:
Should we assert the Heap is deferred in this function?

Reply:
Sounds good.